src/XRP/model/xrp-memo.ts

```diff
@@ -1,4 +1,10 @@
 import { Memo } from '../Generated/web/org/xrpl/rpc/v1/transaction_pb'
+import { stringToUint8Array } from '../../Common/utils'
+
+export interface MemoField {
```

Comment:
can you doc this? It's not immediately obvious what the use is.

Reply:
Sho thang.